workloads/run_workloads.yaml

```
@@ -436,10 +436,17 @@
         own_ip: "{{application_host_ip}}"
         communication_port: 0
       environment: "{{workload_version.env | default({}) }}"
+      block:
+      - name: Mesos | Create stress_ng
```

It shouldn't be this way?
```suggestion
      - name: Mesos | Create tensorflow-benchmark/train workload
```